### PR TITLE
Version 2 of 'Ad feature MPU large' template

### PR DESCRIPTION
**Pull request by tijanamiletic**
Design tweaks to 'Ad feature MPU large' template based on this design:

commercial-item-in-container 1

Old template looks like this:

screen shot 2015-06-03 at 15 03 03

Changes include:
1. Removing padding around image
2. Reducing padding/margin around text
3. Fixing image height to 150px
4. Fixing heading height to allow up to 2 lines of text

### CODE REVIEW

File: static/src/stylesheets/module/commercial/_creatives.scss [hunk 1]
```diff
@@ -647,6 +647,36 @@
...
+
+    &.creative--ad-feature-mpu-large-v2 {
+        padding: 0 0 $gs-baseline / 2;
+
+        .creative__article {
+            padding: 0;
+
+            h2 {
+                padding: $gs-baseline / 2;
+                font-size: 18px;
```
Comment (Calanthe) [inline static/src/stylesheets/module/commercial/_creatives.scss]: you should use https://github.com/guardian/guss-typography to set font values

**Comment by tijanamiletic** [comment]
Font size and line height declaration have been changed to use guss typography

File: static/src/stylesheets/module/commercial/_creatives.scss [hunk 2]
```diff
@@ -647,6 +647,36 @@
         display: block;
         clear: left;
     }
+
+    &.creative--ad-feature-mpu-large-v2 {
```
Comment (uplne) [inline static/src/stylesheets/module/commercial/_creatives.scss]: Do we need to put it inside the `.creative--ad-feature-mpu-large`? It is adding unnecessary nesting/specificity.

**Comment by uplne** [comment]
Other than comments :+1:

**Comment by tijanamiletic** [comment]
Image height now fixed to 156px.

**Pull request merged**